src/Employer/Employer.Web/Views/VacancyManage/EditVacancy.cshtml

```diff
@@ -32,6 +32,6 @@
             <button type="submit" class="govuk-button save-button">Publish</button>
         </form>
         <a asp-show="@Model.HasChanges" asp-route="@RouteNames.CancelVacancyChanges_Get" class="govuk-link das-button-link">Cancel</a>
-        <a asp-show="@Model.IsUnchanged" asp-route="@RouteNames.VacancyManage_Get" class="govuk-button">Return to manage vacancy</a>
+        <a asp-show="@Model.IsUnchanged" asp-route="@RouteNames.VacancyManage_Get" class="govuk-button">Return to manage advert</a>
```

Comment:
There is a mention of vacancy in the EditVacancyPartial. "This vacancy meets disability confident requirements'